interactive/submit.py

```diff
@@ -1,16 +1,15 @@
-import copy
 import os
 import shutil
 import subprocess
 import sys
 import tempfile
-import typing
 from pathlib import Path
 
-from attrs import asdict, define
+from attrs import asdict
 from django.conf import settings
 from django.db import transaction
-from jinja2 import Environment, FileSystemLoader, StrictUndefined
+
+from interactive import AnalysisTemplate
```

Comment:
Can we import from `.` to match all the other imports here?

Reply:
For consistency, sure.

I still maintain importing from . is Just Not Right, but that ship has sailed in job-server :)